app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from os import rename, path, makedirs, replace
+
+
+def move_file(line: str) -> None:
+    line_lst = line.split()
+    if len(line_lst) != 3 or line_lst[0] != "mv":
+        raise ValueError("must be: keyword *mv* and two arguments")
+    line_src = line_lst[1]
+    line_dst = line_lst[2]
```

Comment:
line_src, line_dst what this names mean?
Please choose better names for this variable